test/qa/test_connect.py

```diff
@@ -450,3 +452,25 @@ def test_status_connected(tech, proto, obfuscated):
         assert time_passed - 1 <= time_connected <= time_passed + 1
 
     disconnect_base_test()
+
+
+@pytest.mark.parametrize(("tech", "proto", "obfuscated"), lib.STANDARD_TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
```

Comment:
I think we should not add anymore the retries. If something is broke should stop at the first attempt.
There is also a ticket to remove all this, for all the tests.